### ISSUE TITLE
Ramda does not work with snowpack

### ISSUE THREAD

**Issue by adrianhelvik** [ramda/ramda#3241]
After installing ramda I get the following error in snowpack:

```
[snowpack] Package "ramda" exists but package.json "exports" does not include entry for "./src/forEach".
```

Some dependency of mine is using ramda and I'm trying to locate the issue. Will elaborate here if I find out anything.

**Comment by adrianhelvik** [comment]
This is an issue with the consumer of ramda, not with ramda itself

**Comment by adrianhelvik** [comment]
I did some debugging and the issue was caused by the html-to-react package. It does however use Ramda as you have documented in the readme:

```javascript
import identity from 'ramda/src/identity'

identity()
```

So this is indeed an issue with Ramda and it will be a problem when using Ramda with import maps. Either the documentation should be changed to use .js or the exports-field should be updated to include each sub-package.

```javascript
const fs = require('fs')
const pkg = JSON.parse(fs.readFileSync('./package.json', 'utf8'))
fs.readdirSync('./src').forEach(f => pkg.exports[`./src/${f.replace(/\.js$/, '')]`] = f)
fs.writeFileSync('./package.json', JSON.stringify(pkg, null, 2))
```

Sorry for closing and opening this issue so many times. My brain is a bit fried from trying to migrate from Webpack to a faster bundler.

**Comment by semmel** [comment]
> Either the documentation should be changed ...

```js
import identity from 'ramda/src/identity'
```

is IMO an unfortunate choice to document tree-shaking for Ramda. Regarding the current `"exports"` in `package.json`, that statement violates ES6 module spec and thus gives an `ERR_MODULE_NOT_FOUND` error.

Not being an expert in tree-shaking, I'd vote for changing it to  
```js
import {identity} from 'ramda'; 
```
which works for Rollup.

I guess 
```js
import identity from 'ramda/src/identity.js'
```
would work too, but is too wordy and impractical.

> ... or the exports-field should be updated to include each sub-package

I would not recommend that, because `package.json` needs to be kept human-readable.

**Comment by higherorderfunctor** [comment]
Having a similar issue over at https://github.com/lo1tuma/eslint-plugin-mocha/issues/322 when they upgraded to 0.28.0.

**Comment by CrossEye** [comment]
So is this something we can just fix by updating the README?  Or is there more that we have to do?

**Comment by lo1tuma** [comment]
Isn’t this a duplicate of #3236?
